src/Tests.props

```diff
@@ -6,6 +6,8 @@
     <SingleTestPlatform Condition="'$(SingleTestPlatform)' == ''">false</SingleTestPlatform>
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
     <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);CS8618</NoWarn>
```

Comment:
CS8618 warns when a non-nullable property can contain a null value due to not being initialized in the constructor -- e.g.:

```cs
class Test
{
    public string Abc { get; set; }
}
```

Reply:
Shouldn't they be marked as nullable then?

Reply:
No, it's just an input model.  Finally now we can mark `required`, but that didn't exist in prior versions of .NET.